### PR TITLE
refactor: update check permission messages part 1

### PR DESCRIPTION
**Pull request by SLdragon**

### CODE REVIEW

File: packages/fx-core/src/plugins/solution/fx-solution/solution.ts [hunk 1]
```diff
@@ -1335,30 +1351,51 @@ export class TeamsAppSolution implements Solution {
...
-              }
-            }
-          }
-        }
+      ctx.logProvider?.info("=== M365 account Permissions ===");
+      if (m365PermissionCheckError) {
+        ctx.logProvider?.error(`${errorMsg}: ${m365PermissionCheckError}`);
+      } else {
+        ctx.logProvider?.info(`Your current signed M365 account is: ${userInfo.userPrincipalName}`);
+        this.printPermissionLogs("Azure", results, ctx);
```
Comment (KennethBWSong) [inline packages/fx-core/src/plugins/solution/fx-solution/solution.ts:1359]: Why here is azure?

File: packages/fx-core/src/plugins/solution/fx-solution/solution.ts [hunk 2]
```diff
@@ -1335,30 +1351,51 @@ export class TeamsAppSolution implements Solution {
...
+      }
+
+      ctx.logProvider?.info("=== Azure Resource Permissions ===");
+      if (azurePermissionCheckError) {
+        ctx.logProvider?.error(`${errorMsg}: ${azurePermissionCheckError}`);
+      } else {
+        const accountInfo = await ctx.azureAccountProvider?.getAccountInfo();
+        ctx.logProvider?.info(`Your current signed Azure account is: ${accountInfo!.unique_name}`);
+        this.printPermissionLogs("M365", results, ctx);
```
Comment (KennethBWSong) [inline packages/fx-core/src/plugins/solution/fx-solution/solution.ts:1368]: same as above

File: packages/fx-core/src/plugins/solution/fx-solution/solution.ts (hunk 1 above)
Comment (SLdragon) [reply packages/fx-core/src/plugins/solution/fx-solution/solution.ts:1359]: Oh, sorry for mistake

File: packages/fx-core/src/plugins/solution/fx-solution/solution.ts (hunk 2 above)
Comment (SLdragon) [reply packages/fx-core/src/plugins/solution/fx-solution/solution.ts]: Updated

File: packages/fx-core/src/plugins/solution/fx-solution/solution.ts (hunk 1 above)
Comment (SLdragon) [reply packages/fx-core/src/plugins/solution/fx-solution/solution.ts]: Updated

**Review by KennethBWSong — APPROVED** [review]

**Pull request merged**